src/main/scala/com/fulcrumgenomics/util/UpdateDelimitedFileContigNames.scala

```diff
@@ -99,17 +117,81 @@ class UpdateDelimitedFileContigNames
           case Some(column) =>
             throw new IllegalStateException(s"Did not find contig ${fields(column)} in the given sequence dictionary.")
           case None =>
-            // Write it all out
-            forloop(from = 0, until = numFields) { i =>
-              if (0 < i) out.append(delimiter)
-              out.write(fields(i))
+            // sorting
+            sorter match {
+              case None => // Write out the fields without sorting
+                forloop(from = 0, until = numFields) { i =>
+                  if (0 < i) out.append(delimiter)
+                  out.write(fields(i))
+                }
+                out.write('\n')
+              case Some(_sorter) => // add then to the sorter, to be written later
+                _sorter += CoordinateAndString(
```

Comment:
The name `CoordinateAndString` is confusing.  Why not just call it `LineInfo` or `LineAndSortKeys` or something?

Reply:
The way you're constructing this will use 2x the amount of memory for the sorting.  The whole idea of the `Sorter` is to serialize the whole thing as effectively as possible to reduce storage, and then only hold the key in memory.  Since the key has all the data over again, it'll store twice as much.

Reply:
I _think_ the best way to do this would be to a) just use the line as the thing put into the sorter and b) have the sorter be able to re-split the line and pull out the contig and position fields, but I see that's rather complicating.

Reply:
Fixed